lib/platform_adaptive.dart

```diff
@@ -58,6 +59,26 @@ class PlatformAdaptiveButton extends StatelessWidget {
   }
 }
 
+class PlatformAdaptiveContainer extends StatelessWidget {
```

Comment:
sgtm! and yes less typing. I'm working through developing a list of the small typing snippets we can add for maximal bang for buck.